.github/workflows/analysis.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@3a919529898de77ec3da873e3063ca4b10e7f5cc
         with:
-          version: v1.52.2
+          version: v1.56.2
```

Comment:
the golangci-lint needed to be updated as depguard import rule was enabled and fails without a configuration file - https://github.com/atc0005/go-ci/issues/1024